src/state/slices/portfolioSlice/portfolioSlice.test.ts

```diff
@@ -65,9 +65,6 @@ describe('portfolioSlice', () => {
 
           const portfolio = mockUpsertPortfolio([ethAccount], assetIds)
           store.dispatch(portfolioSlice.actions.upsertPortfolio(portfolio))
-          for (const accountId of portfolio.accounts.ids) {
-            store.dispatch(portfolioSlice.actions.enableAccountId(accountId))
-          }
```

Comment:
These tests require a connected `walletId`. But since these tests are meant to be testing `upsertPortfolio`, I opted to remove the `enableAccountId` action rather than attempt to set the wallet here.